html/navbar.html

```diff
@@ -1,31 +1,50 @@
+      <span class="navbar-toggler-icon"></span>
+  </button>
 
-    <div class="collapse navbar-collapse" id="navbarSupportedContent">
-        <ul class="navbar-nav mr-auto">
-          <li class="nav-item">
-            <a class="nav-link" href="/camera">Camera</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="/disk-memory">Disk &amp; Memory</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="/network">Network</a>
-          </li>
-             <li class="nav-item">
-            <a class="nav-link" href="/wifi-networks">WiFi Networks</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="/speaker">Speaker Test</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="/location">Location</a>
-          </li>
-        </ul>
-    </div>
-    </nav>
+  <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="/camera">Camera</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/disk-memory">Disk &amp; Memory</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/network">Network</a>
+      </li>
+         <li class="nav-item">
+        <a class="nav-link" href="/wifi-networks">WiFi Networks</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/speaker">Speaker Test</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/location">Location</a>
+      </li>
+    </ul>
+    <span class="navbar-text">
+      <svg aria-hidden="true" focusable="false" class="svg-signal" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512">
+        <path class="signal-1" fill="currentColor" 
+        d="M88 384H40c-8.84 0-16 7.16-16 16v96c0 8.84 7.16 16 16 16h48c8.84 0 16-7.16 16-16v-96c0-8.84-7.16-16-16-16z"/>
+        <path class="signal-2" fill="currentColor" 
+        d="M216 288h-48c-8.84 0-16 7.16-16 16v192c0 8.84 7.16 16 16 16h48c8.84 0 16-7.16 16-16V304c0-8.84-7.16-16-16-16z"/>
+        <path class="signal-3" fill="currentColor" 
+        d="M344 192h-48c-8.84 0-16 7.16-16 16v288c0 8.84 7.16 16 16 16h48c8.84 0 16-7.16 16-16V208c0-8.84-7.16-16-16-16z"/>
+        <path class="signal-4" fill="currentColor"
+        d="m472 96h-48c-8.84 0-16 7.16-16 16v384c0 8.84 7.16 16 16 16h48c8.84 0 16-7.16 16-16V112c0-8.84-7.16-16-16-16z"/>
+        <path class="signal-5" fill="currentColor" 
+        d="M600 0h-48c-8.84 0-16 7.16-16 16v480c0 8.84 7.16 16 16 16h48c8.84 0 16-7.16 16-16V16c0-8.84-7.16-16-16-16z"></path>
+      </svg>
+      <div style="display:inline-block">
+        <div id="signal-strength" class="signal-value-div"> - </div> dB
```

Comment:
It just occurred to me that some devices won't have a 3G modem at all. It would be nice to not show anything in the nav bar for this case (or some indicator that there's no 3G modem).

Reply:
Does the `dB` make sense here? There's just a percentage value to work with isn't there?